step-templates/dsc-windows-service.json

```diff
@@ -0,0 +1,58 @@
+  },
+  "SensitiveProperties": {},
+  "Parameters": [
+    {
+      "Name": "Services",
+      "Label": "Services to align",
+      "HelpText": "A comma delimited list of services or wildcards (eg. \"Sky\") to align",
+      "DefaultValue": null,
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Name": "ServicesToIgnore",
+      "Label": "Services to ignore",
+      "HelpText": "A comma delimited list of services or wildcards (eg. \"Sky\") to ignore",
+      "DefaultValue": null,
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Name": "DesiredState",
+      "Label": "Desired state",
+      "HelpText": "The desired state of the service/s. Specify either \"Started\" or \"Stopped\"",
+      "DefaultValue": null,
+      "DisplaySettings": {
+        "Octopus.ControlType": "Select",
+        "Octopus.SelectOptions": "Running|Started\nStopped|Stopped"
+      }
+    },
+    {
+      "Name": "TimeoutSeconds",
+      "Label": "Timeout in seconds",
+      "HelpText": "The number of seconds to wait for the service/s to align before timing out and throwing an exception",
+      "DefaultValue": "300",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    }
+  ],
+  "LastModifiedOn": "2017-06-20T14:00:00.000+00:00",
+  "LastModifiedBy": "alexhildyard",
```

Comment:
If you could update this to **dunedinsoftware** then the author link on the Library site will link correctly to your :octocat: profile